q2cli/handlers.py

```diff
@@ -351,20 +414,22 @@ def __init__(self, name, default=NoDefault, description=None):
     def get_click_options(self):
         import click
         import q2cli
+        import q2cli.core
 
         name = '--' + self.cli_name
         type = click.Path(exists=True, file_okay=True, dir_okay=False,
                           readable=True)
+        type = q2cli.core.MultipleType(type)
         help = ('Metadata file or artifact viewable as metadata. This '
-                'option may be supplied multiple times to merge metadata')
+                'option may be supplied multiple times to merge metadata.')
```

Comment:
There is a missing space after this sentence, for example:

```
  --m-sample-metadata-file MULTIPLE PATH
                                  Metadata file or artifact viewable as
                                  metadata. This option may be supplied
                                  multiple times to merge metadata.The sample
                                  metadata.  [optional]
```

Also, there are two spaces between the param description and the `[optional]` tag.

Reply:
oops!